docs/tutorial-get.rst

```diff
@@ -63,7 +63,7 @@ But the real power of the ``filter()`` method is in combining multiple
 parameters:
 
 .. literalinclude:: files/tutorial-get.py
-  :lines: 20-27
+  :lines: 20-29
```

Comment:
Does this also break all other references further down the file?

Reply:
Yes :-)